app/sidekiq/evss/disability_compensation_form/submit_form526.rb

```diff
@@ -107,7 +107,7 @@ def perform(submission_id) # rubocop:disable Metrics/MethodLength
           begin
             submit_to_claims_api = submission.claims_api?
             # send submission data to either EVSS or Lighthouse (LH)
-            response = if Flipper.enabled?(:disability_compensation_lighthouse_submit_migration, user) ||
+            response = if Flipper.enabled?(:disability_526_toxic_exposure, user) ||
```

Comment:
I think this could be further consolidated by also removing the Flipper evaluation. If the corresponding feature was ever enabled for a user, their associated in progress form data would be appended with the `includeToxicExposure` indicator, which signifies the setting of the `submit_endpoint` to 'claims_api' when the submission is created.

Reply:
agreed. When I proposed the `includeToxicExposure` element,  the expectation was that the flipper would be used in the IPF creation, and then the `includeToxicExposure` indicator would be used going forward.

Reply:
Your collective confidence is inspiring- I have removed it! Please take a quick look (seems like I need re-approval anyway. Thanks! @freeheeling @tblackwe @aurora-a-k-a-lightning